static/admin/config.yml

```diff
@@ -29,6 +29,34 @@ collections:
             fields:
               - {label: "Title", name: "title", widget: "string"}
               - {label: "Body", name: "body", widget: "markdown", required: false}
+          - label: "Team Section"
+            name: "team_section"
+            widget: "object"
+            fields:
+              - {label: "Title", name: "title", widget: "string"}
+              - {
+                label: "Team",
+                name: "team",
+                widget: "list",
+                allow_add: true,
+                fields:
+                  - {label: "Name", name: "name", widget: "string"}
+                  - {label: "Title", name: "title", widget: "string"}
+                  - {
+                      label: "Profile Image",
+                      name: "image",
+                      widget: "image",
+                      hint: "Image should be at least the 2x resolution size",
```

Comment:
Might be worth noting what this is? Is it 1024px?

Reply:
Updated!